isaric/parsers/ccpuk_observations.toml

```diff
@@ -0,0 +1,465 @@
+phase = "admission"
+is_present = { field = "cough_ceoccur_v2", ref = "Y/N/NK" }
+if = { bleed_ceoccur_v2 = { "!=" = 3} }
+#description = "Bleeding (Haemorrhage)"
+
+[[observation]]
+name = "cough"
+date = { ref = "observationDateHierarchy" }
+phase = "admission"
+is_present = { if field = "cough_ceoccur_v2", ref = "Y/N/NK" }
+if = { cough_ceoccur_v2 = { "!=" = 3} }
+#description = "Cough"
+
+[[observation]]
+name = "cough"
+date = { ref = "followupDateHierarchy" }
+phase = "followup"
+is_present = { if field = "flw_cough", values = { 1 = true, 0 = false } }
+if = { flw_cough = { "!=" = 99} }
+#description = "Cough"
+
+[[observation]]
+name = "avpu"
+date = { field = "daily_dsstdat", description = "Date of assessment" }
+phase = "study"
+value_char = { field = "avpu_vsorres", values = { 1 = "Alert", 2 = "Verbal", 3 = "Pain", 4 = "Unresponsive" }}
+if = { avpu_vsorres = { "!=" = 5} }
+
+[[observation]]
+name = "oxygen_saturation_percent"
+phase = "admission"
+date = { field = "hostdat", description = "Admission date at this facility" } # USE one of the defs above?
+context = { field = "oxy_vsorresu", description = "Oxygen saturation on:", values = { 1 = ["Room air"], 2 = ["Oxygen therapy"]} }
+value_num = { field = "oxy_vsorres" }
+if = { oxy_vsyn = 1 }
+
+[[observation]]
+name = "glasgow_coma_score"
+phase = "study"
+date = { field = "daily_dsstdat", description = "Date of assessment" }
+value_num = { field = "daily_gcs_vsorres" }
+if = { daily_gcs_lbyn = 1 }
+#description = "Glasgow Coma Score (GCS/15)"
+
+[[observation]]
+name = "conjunctivitis"
+phase = "admission"
+date = { ref = admissionDateHierarchy }
+is_present = { if field = "conjunct_ceoccur_v2", ref = "Y/N/NK" }
+if = { conjunct_ceoccur_v2 = { "!=" = 3} }
+
+[[observation]]
+name = "malnutrition"
+phase = "admission"
+date = { ref = admissionDateHierarchy }
+is_present = { if field = "malnutrition_mhyn", ref = "Y/N/NK" }
+if = { malnutrition_mhyn = { "!=" = 3} }
+
+[[observation]]
+name = "rhinorrhoea"
+phase = "admission"
+date = { ref = admissionDateHierarchy }
+is_present = { if field = "runnynose_ceoccur_v2", ref = "Y/N/NK" }
+if = { runnynose_ceoccur_v2 = { "!=" = 3} }
+
+[[observation]]
+name = "seizures"
+phase = "admission"
+date = { ref = admissionDateHierarchy }
+is_present = { if field = "seizures_cecoccur_v2", ref = "Y/N/NK" }
+if = { seizures_cecoccur_v2 = { "!=" = 3} }
+
+[[observation]]
+name = "sore_throat"
+phase = "admission"
+date = { ref = admissionDateHierarchy }
+is_present = { if field = "sorethroat_ceoccur_v2", ref = "Y/N/NK" }
+if = { sorethroat_ceoccur_v2 = { "!=" = 3} }
+
+[[observation]]
+name = "vomiting"
+phase = "admission"
+date = { ref = admissionDateHierarchy }
+is_present = { if field = "vomit_ceoccur_v2", ref = "Y/N/NK" }
+if = { vomit_ceoccur_v2 = { "!=" = 3} }
+
+[[observation]]
+name = "vomiting"
+phase = "followup"
+date = { ref = followupDateHierarchy }
+is_present = { if field = "flw_nausea", values = { 1 = true, 0 = false } }
```

Comment:
if not valid here, only `{ field = "flw_nausea", ... }`

Reply:
not sure I understand your comment/suggestion. we'll come back to this..